sys/net/gnrc/conn/gnrc_conn.c

```diff
@@ -52,11 +52,33 @@ static inline void _srcport(uint16_t *port, gnrc_pktsnip_t *hdr)
 int gnrc_conn_recvfrom(conn_t *conn, void *data, size_t max_len, void *addr, size_t *addr_len,
                        uint16_t *port)
 {
+    return gnrc_conn_recvfrom_timeout(conn, data, max_len, addr, addr_len, port, CONN_NO_TIMEOUT);
```

Comment:
The idea here was to avoid load on the xtimers linked list and all it's calculations if there is no timeout specified. This could be moved to the xtimer driver though. And the adding and subtracting is done because there is a loop and I would like the timeout value to be hard. Although, this might never be the case if an different message is received.